tests/boot/boot_from_pxe.pm

```diff
@@ -14,7 +14,7 @@ use base "opensusebasetest";
 use testapi;
 
 sub run() {
-    check_screen("virttest-bootloader", 60);
+    check_screen("virttest-bootloader", 20);
```

Comment:
It is more robust and also saves time if you replace the check_screen by an assert_screen with a list of all needles you can assume at this time and then check with `match_has_tag`. `match_has_tag` is already used afterwards so I don't see a problem with that.